test/generateControllers.test.js

```diff
@@ -146,3 +150,243 @@ describe('Herbs2Rest - Generate Routes With Herbarium', () => {
     assert.rejects(() => generateRoutes(null, routes))
   })
 })
+
+describe('Herbs2Rest - Generate Custom Routes With Herbarium', () => {
+  const Test =
+    entity('Test', {
+      id: field(Number, { isId: true })
+    })
+
+  const crudOperation = (param) => () => usecase(`${param} Usecase`, {
+    request: {},
+    authorize: async _ => Ok(),
+    'Test step': step(_ => Ok())
+  })
+
+  const usecaseTest = () => {
+    herbarium.requireAll({})
+
+    herbarium.entities
+      .add(Test, 'Test')
+
+    herbarium.usecases
+      .add(crudOperation('Read'), 'ReadUsecase')
+      .metadata({
+        group: 'Test', operation: herbarium.crud.read, entity: Test,
+        REST: {
+          path: '/customgetbyid/:id',
+        }
+      })
+
+    herbarium.usecases
+      .add(crudOperation('ReadAll'), 'ReadAllUsecase')
+      .metadata({
+        group: 'Test', operation: herbarium.crud.readAll, entity: Test,
+        REST: {
+          path: '/customget',
+        }
+      })
+
+    herbarium.usecases
+      .add(crudOperation('Create'), 'CreateUsecase')
+      .metadata({ group: 'Test', operation: herbarium.crud.create, entity: Test })
+
+    herbarium.usecases
+      .add(crudOperation('Update'), 'UpdateUsecase')
+      .metadata({
+        group: 'Test', operation: herbarium.crud.update, entity: Test, REST: {
+          path: '/customput'
+        }
+      })
+
+    herbarium.usecases
+      .add(crudOperation('Delete'), 'DeleteUsecase')
+      .metadata({
+        group: 'Test', operation: herbarium.crud.delete, entity: Test,
+        REST: {
+          path: '/customdeletebyid/:id'
+        }
+      })
+  }
+
+  beforeEach(() => usecaseTest())
+
+  it('Should resolve and create a custom get by id route after add inside Herbarium usecase list', (done) => {
```

Comment:
this is a test for a path override, right?

suggestion:
```js
describe('Generate Custom Routes With Herbarium', ()
   it('get - should ignore the conventions and use the given path'
```

Reply:
I Agree